python/tank/authentication/interactive_authentication.py

```diff
@@ -198,10 +212,10 @@ def renew_session(user):
                                      this exception is raised.
     """
     logger.debug("Credentials were out of date, renewing them.")
-    QtCore, QtGui, has_ui = _get_qt_state()
+    QtCore, QtGui, QtNetwork, QtWebKit, has_ui = _get_qt_state()
     # If we have a gui, we need gui based authentication
     if has_ui:
-        authenticator = UiAuthenticationHandler(is_session_renewal=True)
+        authenticator = UiAuthenticationHandler(is_session_renewal=True, cookies=user.get_cookies())
```

Comment:
It would be a bit clunky to go about this way. We're not passing in the user because host, login and proxy may or may not be available values when you are already logged in. Maybe passing in the struct you are mentioning below would be a better refactor?